docs/monorepo.md

```diff
@@ -211,7 +217,7 @@ There's no need to be concerned about collisions with other projects' coverage f
 
 Most projects in the monorepo should have a mirror repository holding a built version of the project, ready for deployment. Follow these steps to create the mirror repo and configure the monorepo tooling to push to it.
 
-1. Create the mirror repo on GitHub. It will most likely be named like "https://github.com/Automattic/jetpack-_something_".
+1. Create the mirror repo on GitHub. It will most likely be named like "<https://github.com/Automattic/jetpack>-_something_".
```

Comment:
On the other hand, you kind of fixed this line. But an even better fix would be to disable the link entirely, maybe something like
```suggestion
1. Create the mirror repo on GitHub. It will most likely be named like "https://<span></span>github.com/Automattic/jetpack-_something_".
```
Same on original-side line 110. But maybe we should leave all that cleanup for a separate PR.